specifyweb/stored_queries/build_models.py

```diff
@@ -78,11 +76,11 @@ def make_relationship(reldef):
             column = getattr(table.c, reldef.column)
 
             relationship_args = {'foreign_keys': column}
+            if remote_class is cls:
+                relationship_args['remote_side'] = table.c[ tabledef.idColumn ]
```

Comment:
I will have to read up a little more on the sqlalchemy docs to completely understand everything that is going on here, but when compared to how it was previously,  this look alright!